automation/check-patch.e2e-kubemacpool-functests.sh

```diff
@@ -28,9 +28,27 @@ main() {
     echo "Deploy KubeVirt latest stable release"
     ./hack/deploy-kubevirt.sh
 
+    echo "Get skip tests regex"
+    SKIPPED_TESTS=$(__get_skipped_tests)
+
     # Run KubeMacPool functional tests
     cd ${TMP_COMPONENT_PATH}
-    KUBECONFIG=${KUBECONFIG} E2E_TEST_ARGS="--junit-output=$ARTIFACTS/junit.functest.xml" make functest
+
+    KUBECONFIG=${KUBECONFIG} E2E_TEST_ARGS="--junit-output=$ARTIFACTS/junit.functest.xml --ginkgo.skip $SKIPPED_TESTS" make functest
+}
+
+function __get_skipped_tests() {
+	local skipped_regex=""
+
+	if [ ! -z "$(grep 'default/mutatevirtualmachines_opt_in_patch.yaml' hack/components/bump-kubemacpool.sh)" ]; then
```

Comment:
this is checking opt-out to skip opt-in so 

```bash
if grep 'default/mutatevirtualmachines_opt_out_patch.yaml' hack/components/bump-kubemacpool.sh; then
```
```

Reply:
notice the `!`
I am actually asking: if `default/mutatevirtualmachines_opt_out_patch.yaml` is mentioned on `hack/components/bump-kubemacpool.sh` then...

Reply:
wow great catch!!
Done